components/ide/jetbrains/backend-plugin/build.gradle-latest.kts

```diff
@@ -0,0 +1,163 @@
+// Copyright (c) 2024 Gitpod GmbH. All rights reserved.
+// Licensed under the GNU Affero General Public License (AGPL).
+// See License.AGPL.txt in the project root for license information.
+
+import io.gitlab.arturbosch.detekt.Detekt
```

Comment:
I'm not sure we need this one.

We can use build.gradle.kts for latest, and build.gradle-stable.kts.

If someone wants to change they can unload one from IJ and load another.

for building we can pass different with ./gradlew --build-file

Reply:
Build failed with this change https://github.com/gitpod-io/gitpod/pull/19978/commits/3cc44942122ec6ce69ae82db3812a633f4fee431

Searched  error `Build cache controller already set` that it may need `--no-build-cache`, but after it, it failed again (if we copy it into build.gradle.kts, it will work): 
<img width="1599" alt="image" src="https://github.com/gitpod-io/gitpod/assets/20944364/6ab12a68-88ba-411e-aead-b4bd1add5edd">

Reply:
I'm going to revert that commit

Reply:
Base on the reverting, keep different `build.gradle-<version>.kts` is more useful for me: so that I could keep these two copies and use `build.gradle.kts` to debug with `launcher-dev-server.sh`